docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss

```diff
@@ -0,0 +1,159 @@
+$hb-sidebar-width: 16.5rem;
+
+.hb-three-column-w-image,
+.hb-three-column {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1rem 3rem;
+  // Merani shows all gutter widths as 3rems at all screen sizes. Let me know what
+  // people think
+
+  @include grid-media('lg') {
+    flex-wrap: nowrap;
+  }
+
+  @include grid-media('2xl') {
+    margin: 1rem auto;
+  }
+
+  &__sidebar-1 {
+    flex: 1 1 100%;
+    margin: 0 0 3rem 0;
+
+    @include grid-media('md') {
+      flex: 0 0 $hb-sidebar-width;
+      margin: 0 3rem 0 0;
+    }
+  }
+
+  &__main {
+    flex: 1 1 100%;
+    margin-bottom: 3rem;
+
+    @include grid-media('md') {
+      flex: 0 0 calc(100% - #{$hb-sidebar-width} - #{$hb-gutter-width});
+      margin-bottom: 0;
+
+      .hb-three-column-w-image--no-sidebar &,
+      .hb-three-column--no-sidebar & {
+        flex: 1 1 100%;
+      }
+    }
+
+    @include grid-media('lg') {
+      flex: 1 1 100%;
+    }
+
+    .hb-three-column-w-image--one-sidebar &,
+    .hb-three-column--one-sidebar & {
+      @include grid-media('2xl') {
+        max-width: 68.75rem;
+      }
+    }
+  }
+
+  &__main-top {
+    // should this style apply to the 3 column grid?
+    // need to make sure I didnt mess up node news details page
```

Comment:
Yeah, we're affecting the three col w/ image. I would think we would want those layouts to match for the most part so maybe pair with Merani on that?

Reply:
Yeah we'll keep the styles separate for now